packages/core/addon/components/navi-visualization-config/series-chart.ts

```diff
@@ -13,54 +13,74 @@
 
 import { assign } from '@ember/polyfills';
 import { A as arr } from '@ember/array';
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import { set, get, computed, action } from '@ember/object';
 import { readOnly } from '@ember/object/computed';
 import { isArray } from '@ember/array';
+//@ts-ignore
 import { copy } from 'ember-copy';
 import { dataByDimensions } from 'navi-core/utils/data';
-import layout from '../../templates/components/navi-visualization-config/series-chart';
 import { values, reject } from 'lodash-es';
-import { layout as templateLayout, tagName } from '@ember-decorators/component';
-
-@templateLayout(layout)
-@tagName('')
-class NaviVisualizationConfigSeriesChartComponent extends Component {
+import RequestFragment from '../../models/bard-request-v2/request';
+import { ResponseV1 } from 'navi-data/addon/serializers/facts/interface';
+import ColumnFragment from '../../models/bard-request-v2/fragments/column';
+
+type Args = {
+  request: RequestFragment;
+  response: ResponseV1;
+  seriesConfig: TODO;
```

Comment:
Since this component applies to multiple series types and I haven't gotten to `metric` or `dateTime` series yet, I'll just change `object` to `TODO` for now.